src/main/java/vazkii/zetaimplforge/ForgeZeta.java

```diff
@@ -212,6 +205,35 @@ public void lootTableLoad(LootTableLoadEvent e) {
 		playBus.fire(new ForgeZLootTableLoad(e), ZLootTableLoad.class);
 	}
 
+	public void livingConversion(LivingConversionEvent e) {
+		playBus.fire(new ForgeZLivingConversion(e), ZLivingConversion.class);
+	}
+
+	public void livingConversionPost(LivingConversionEvent.Post e) {
+		playBus.fire(new ForgeZLivingConversion.Post(e), ZLivingConversion.Post.class);
+	}
+
+	public void anvilUpdate(AnvilUpdateEvent e) {
+		playBus.fire(new ForgeZAnvilUpdate(e), ZAnvilUpdate.class);
+	}
+
+	public void anvilUpdateLowest(AnvilUpdateEvent e) {
+		playBus.fire(new ForgeZAnvilUpdate.Lowest(e), ZAnvilUpdate.Lowest.class);
+	}
+
+	public void entityMobGriefing(EntityMobGriefingEvent e) {
+		playBus.fire(new ForgeZEntityMobGriefing(e), ZEntityMobGriefing.class);
+	}
+
+	public void livingDrops(LivingDropsEvent e) {
+		playBus.fire(new ForgeZLivingDrops(e), ZLivingDrops.class);
+	}
+
+	public void inputMouseButton(InputEvent.MouseButton e) {
```

Comment:
This event should probably go in ForgeZetaClient for side-safety reasons.

Reply:
Yeah I noticed that earlier I'll do the change once I'm home